guides/dev/deep_dive/errors/README.md

```diff
@@ -37,6 +37,11 @@ This article contains the list of CKEditor error codes and their explanation. Th
 	* `plugin`: The name of the plugin that cannot be removed.
 	* `requiredBy`: The name of the plugin whose requirements block the removal.
 
+## embed-no-provider-url
+* Location: `plugins/embed/plugin.js`
+* Description: No embed provider URL configured. Since 4.6.0 this value is empty by default.
```

Comment:
There needs to be a link to more information. For sure we need to link to http://docs.ckeditor.com/#!/api/CKEDITOR.config-cfg-embed_provider